.changes/0.0.0.md

```diff
@@ -0,0 +1,3 @@
+## Previous Releases
+
+For information on prior major and minor releases, see their changelogs:
```

Comment:
This is where the links to the previous versions should live.  They will get overwritten if you only put them into the `CHANGELOG.md` file.

Every time `changie merge` is run it overwrites the entire contents of the `CHANGELOG.md` by stiching together the .`md` files in `/.changes` with the header file.

Reply:
moved release information to 0.0.0.md tried playing around with reformatting the file name didn't really find any success here (changie.yml doesn't hard ref the footer file and there is no config for footerPath so likely falls in as part of merge action).